src/router.jsx

```diff
@@ -0,0 +1,53 @@
+import { createBrowserRouter } from "react-router-dom";
+import App from "./App";
+import Login from "./pages/Login";
+import ResetPassword from "./pages/ResetPassword";
+import SignUp from "./pages/SignUp";
+import TimeTracker from "./pages/TimeTracker";
+import Projects from "./pages/Projects";
+import Users from "./pages/Users";
+import ConfirmResetPassword from "./pages/ConfirmResetPassword";
+
+export default createBrowserRouter([
+  {
+    path: "/",
+    element: <App />,
+    errorElement: <div>Error Occurrec</div>,
+    children: [
+      {
+        errorElement: <div>test</div>,
+        children: [
+          {
+            index: true,
+            element: <TimeTracker />,
+          },
+          {
+            path: "/projects",
+            element: <Projects />,
+          },
+          {
+            path: "/users",
+            element: <Users />,
+          },
+          {
+            path: "/login",
+            element: <Login />,
+          },
+          {
+            path: "/signup",
+            element: <SignUp />,
+          },
+          {
+            path: "/resetpassword",
+            element: <ResetPassword />,
+          },
+          {
+            path: "/confirmresetpassword",
```

Comment:
(comment): Typically, we will use either title case or kebab case for routes, especially with multiple names. So, in this case, we would use `confirm-reset-password`, to aid in readability.